src/DebuggerData.h

```diff
@@ -19,58 +19,62 @@ struct MemoryLayout
 	int mapperSize[4][4];
 };
 
+struct Breakpoint;
```

Comment:
Perfectly fine.
I personally would have kept `Breakpoint` a nested class inside `BreakPoints`, but this is fine as well.